autocomplete/definitions/global/tes3ui/menuMode.lua

```diff
@@ -1,6 +1,7 @@
 return {
 	type = "function",
 	description = [[Checks if the game is in menu mode.]],
-	returnDescription = "true if in menu mode.",
-	valuetype = "boolean",
+	returns = {
+		{ name = "result", type = "boolean", description = "true if in menu mode." }
```

Comment:
A small stylistic nitpick: could you either capitalize the first letter in the description or put true in backticks?